src/api-request-handler.ts

```diff
@@ -63,6 +65,10 @@ export class ApiRequestHandler {
             switch (error.response.status) {
                 case 400:
                     throw new InvalidParameterError(error);
+                case 403:
+                    throw new ForbiddenAccessError(error);
+                case 404:
```

Comment:
Where is the 403 status code documented on docs.rev.ai?

I don't see anything on https://docs.rev.ai/api/asynchronous/reference/#operation/GetJobById


Also, 404 is an expected status code for GetJobById